src/commands/new.ts

```diff
@@ -202,6 +202,10 @@ export default {
       // rename the babel.config.expo.js
       filesystem.rename("./babel.config.expo.js", "babel.config.js")
 
+      // remove the metro config js, which causes problems with publishing
+      // see: https://github.com/infinitered/ignite/issues/1904
+      filesystem.remove("./metro.config.js")
```

Comment:
Expo apps by default do have a metro config file, so I would think we'd want to preserve that instead of removing this? Looks like this is the template for it: https://github.com/expo/expo/blob/168ee43f71f005baa11edf98e518593443e1807a/templates/expo-template-bare-minimum/metro.config.js

Reply:
Good point @lindboe -- just pushed up a change to address this.